data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg

```diff
@@ -0,0 +1,948 @@
+        [+unit]
+            [modifications]
+                {TRAIT_RESILIENT}
+            [/modifications]
+        [/unit]
+
+        [+unit]
+            [modifications]
+                {TRAIT_INTELLIGENT}
+            [/modifications]
+        [/unit]
+
+        {NOTRAIT_UNIT       5 (Woodsman) 17 26} {FACING se}
+        [+unit]
+            [modifications]
+                {TRAIT_STRONG}
+            [/modifications]
+        [/unit]
+
+        [+unit]
+            [modifications]
+                {TRAIT_RESILIENT}
+            [/modifications]
+        [/unit]
+    [/side]
+    # peasants escape when they get to the west edge
+    [event]
+        name=moveto
+        first_time_only=no
+
+        [filter]
+            side,x=5,1
+        [/filter]
+
+        {KILL id=$unit.id}
+    [/event]
+
+    [event]
+        name=prestart
+
+        # give the undead nearby villages, so they don't waste their tiny starting army capturing them
+        {CAPTURE_VILLAGES 2 29 5  5}
+        {CAPTURE_VILLAGES 2 17 1  1} # this one has a vampire bat on it
+        {CAPTURE_VILLAGES 3 35 15 3}
+        {CAPTURE_VILLAGES 4 30 25 7}
+
+        # need to set recruits per-unit (instead of per-side) so we don't mess up the plague staff
+        [set_extra_recruit]
+            id=Gweddry
+            extra_recruit={EARLY_RECRUIT_LIST}
+        [/set_extra_recruit]
+
+        {PLACE_IMAGE scenery/trapdoor-closed.png 10 15}
+
+        [micro_ai]
+            ai_type=coward
+            action=add
+            side=5
+            [filter]
+                side=5
+            [/filter]
+            distance=99
+            seek_x,seek_y=1,6
+            attack_if_trapped=yes
+        [/micro_ai]
+
+        [objectives]
+            side=1
+            [objective]
+                description= _ "Survive until turns run out,
+    then move any unit just outside your keep, to 10,15."
+                condition=win
+                [show_if]
+                    [variable]
+                        name=trapdoor_open
+                        not_equals=yes
+                    [/variable]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Move any unit to the trapdoor just outside your keep (10,15)"
+                condition=win
+                [show_if]
+                    [variable]
+                        name=trapdoor_open
+                        equals=yes
+                    [/variable]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Death of Gweddry or Dacyn"
+                condition=lose
+            [/objective]
+            [gold_carryover]
+                carryover_percentage=40
+            [/gold_carryover]
+        [/objectives]
+    [/event]
+
+    [event]
+        name=start
+        [message]
+            speaker=Gweddry
+            message= _ "Watchmen, report. What’s going on?"
```

Comment:
Are there supposed to be multiple watchmen addressed here? Only one speaks. I haven't looked at the scenario to see how many units are standing by.

Reply:
Yeah, there's 2 peasants and a woodsman standing around.  It's ambiguous if they're all watchmen or if some are just farmers, so if you want to change it to watchman that'll work fine as well.

Reply:
I would say it's up to the author in this case. Changing it to "watchman" gives the implication that Gweddry doesn't consider the others to be watchmen. Gweddry's assumption then informs the player's viewpoint of whether they are, in fact, watchmen.

So basically, don't change it if the intent is to give the implication that it's a group of watchmen, rather than a single watchman and a few civilians.

Reply:
I was considering that, even if all three are supposed to be standing on watch, Gweddry could just be addressing the most senior of them. Again, it comes down to the intention - I don't really mind what Gweddry says so long as it aligns with was intended.